src/main/java/com/pace/hibernate/model/Student.java

```diff
@@ -0,0 +1,27 @@
+package com.pace.hibernate.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "students")
+public class Student {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private int id;
+    @Column(name = "name")
+    private String name;
+    @Column(name = "email")

```

Comment:
learning >> try to add some regex for email as well as add phno and add regex for that too